components/style/values/specified/list.rs

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! `list` specified values.
+
+use cssparser::{Parser, Token, serialize_string};
+use parser::{Parse, ParserContext};
+use std::fmt;
+use style_traits::{ParseError, StyleParseErrorKind, ToCss};
+
+/// Specified and computed `quote` property
+#[derive(Clone, Debug, MallocSizeOf, PartialEq, ToComputedValue)]
+pub struct Quotes(pub Vec<(String, String)>);
+
+impl ToCss for Quotes {
+    fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
+        if self.0.is_empty() {
+            return dest.write_str("none")
+        }
+
+        let mut first = true;
+        for pair in &self.0 {
```

Comment:
nit: This could read nicer, maybe with `for &(ref first, ref second) in ..`

Also I believe we have helpers to serialize this in a nicer way, and even `derive(..)` it? I think to recall `SequenceWriter` and `css(comma)`.

Maybe converting the `(String, String)` into something like:

```rust
struct QuotePair {
    first: String,
    second: String,
}
```

You could derive this if tuples don't work... But not sure if worth though.